docs/content/CONTRIBUTING.md

```diff
@@ -17,9 +17,16 @@ contribution. See the [DCO](https://github.com/kcp-dev/kcp/tree/main/DCO) file f
 ### Prerequisites
 
 1. Clone this repository.
-2. [Install Go](https://golang.org/doc/install) (1.19+).
+2. [Install Go](https://golang.org/doc/install) (currently 1.19).
 3. Install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl).
 
+Please note that the go language version numbers in these files must exactly agree:
+    -  Your local go/go.mod file, kcp/.ci-operator.yaml, kcp/Dockerfile, and in all the kcp/.github/workflows yaml files that specify go-version.
+    In kcp/.ci-operator.yaml the go version is indicated by the "tag" attribute.
+    In kcp/Dockerfile it is indicated by the "golang" attribute
+    In go.mod it is indicated by the "go" directive."
+    In the .github/workflows yaml files it is indicated by "go-version"
```

Comment:
Thanks for this! Could you please update the formatting so it renders better when the markdown is converted to html? See https://github.com/kcp-dev/kcp/blob/d5ad69188d61b6714f4181bd1119dba23a541aed/docs/content/CONTRIBUTING.md for example:

![image](https://user-images.githubusercontent.com/354/229888199-7562c221-9a46-4659-b527-a783f0a20ce8.png)

Reply:
i think I made the change. Let me know if there are still issues.